webrtc-stats.html

```diff
@@ -768,6 +768,7 @@
              double             burstDiscardRate;
              double             gapLossRate;
              double             gapDiscardRate;
+             DOMHighResTimeStamp  lastPacketReceivedTimestamp;
```

Comment:
I see, @jan-ivar I could move the timestamp to the appropriate inbbound/outbound local objects, that might seem cleaner. WDYT?